### ISSUE TITLE
add ability to plot sample output_type data

### ISSUE THREAD

**Issue by nickreich** [Infectious-Disease-Modeling-Hubs/hubVis#31]
Currently, a limitation of plot_step_ahead_model_data() is that [it only accepts median and quantile output_types](https://github.com/Infectious-Disease-Modeling-Hubs/hubVis/blob/9efc5420a2806929454f8b6b92d00952ea444dd3/R/plot_step_ahead_model_output.R#L133). 

We should support the plotting of sample output_type data by inferring quantiles from it. Currently, a user would have to manually transform the data to quantiles. Possible approach: add sample output_type to the list of valid types and make [the appropriate transformation](https://github.com/Infectious-Disease-Modeling-Hubs/hubData/issues/8) to quantile output_type data. 

Acceptance criteria:
 - a user passes sample output_type data to plot_step_ahead_model_output() and gets a plot returned, with intervals if specified

**Comment by LucieContamin** [comment]
Would it make sense to have a choice to either:
- calculate the quantiles and return a plot with intervals as specified
- return a "spaghetti plot" of the samples

**Comment by elray1** [comment]
i like that, and would add "or both"

**Comment by elray1** [comment]
yeah, I was thinking both on the same plot.  i've seen this done before and found it helpful, but agree with your comment that it could get crowded.  thinking about it now, i think the examples I saw showed only a few of the samples.

**Comment by LucieContamin** [comment]
Yes, I was thinking the same for the samples, we can use point or line depending if they are trajectories or not. I like it and should not be difficult to implement

**Comment by elray1** [comment]
this point relates to https://github.com/Infectious-Disease-Modeling-Hubs/hubData/issues/6 as well
